src/powerquery-language-services/inspection/task.ts

```diff
@@ -53,12 +54,18 @@ class InspectionInstance implements Inspected {
             const allIdentifierIdSet: Set<number> =
                 this.nodeIdMapCollection.idsByNodeKind.get(Ast.NodeKind.Identifier) || new Set();
```

Comment:
nit: use the null coalescing operator `??` instead of or operator `||` when doing a `collection.get`.